src/admin-views/widgets/list.php

```diff
@@ -23,6 +23,39 @@
 }
 
 foreach ( $admin_fields as $field_id => $field ) {
+	// Handle a section start. May contain a section title.
+	if ( false !== stripos( $field_id, 'section_start' ) ) {
+		$section_classes = [ 'tribe-events-widget-admin-form__input-section' ];
+		if ( ! empty( $field['classes'] ) ) {
+			$classes = $field['classes'];
+
+			if ( ! is_array( $classes ) ) {
+				$classes = explode( ' ', $classes );
+			}
+
+			$section_classes = array_merge( $section_classes, $classes );
+		}
+
+		?>
+		<div <?php tribe_classes( $section_classes ); ?>>
+			<?php
+			if ( ! empty( $field['title'] ) ) {
+				// Note: the actual widget title/handle is an <h3>.
+				?>
+				<h4><?php echo esc_html( $field['title'] ); ?></h4>
+				<?php
+			}
+
+			continue;
+	}
+
+	// Handle a section end.
+	if ( false !== stripos( $field_id, 'section_end' ) ) {
```

Comment:
Since we're trying to detect if the `$field_id` **starts** with `section_end` I would use `0 === stripos` here.